pkg/driver/controller.go

```diff
@@ -133,7 +133,8 @@ func (cs *controller) init() error {
 	if zfs.GoogleAnalyticsEnabled == "true" {
 		analytics.RegisterVersionGetter(version.GetVersionDetails)
 		analytics.New().CommonBuild(DefaultCASType).InstallBuilder(true).Send()
-		go analytics.PingCheck(DefaultCASType, Ping)
+		go analytics.PingCheck(DefaultCASType, Ping, false)
```

Comment:
[This](https://github.com/openebs/google-analytics-4/blob/develop/usage/ping.go#L39) is the definition for the PingCheck function after the changes of https://github.com/openebs/google-analytics-4/pull/8.

The true/false is to trigger an immediate ping. With heartbeat, we'd want it trigger immediately and then trigger as per the ticker schedule, so that gets a `true`. With ping, we'd want it to avoid pinging immediately and just stick to the ticker schedule, so it gets a false. It's largely a lazy extension of the already existing PingCheck API.
```go
if pingImmediately {
	// Ping immediately.
	pingSender.Send()
}
```